### ISSUE TITLE
New version 1.118 has changed API things. Broke this fine tool 

### ISSUE THREAD

**Issue by nsrafk** [Salvoxia/immich-folder-album-creator#71]
As the title. Breaking changes in the new update released today.

I hope its easy to fix! Good luck :D

**Comment by Salvoxia** [comment]
Thanks for the heads-up. Will take a look this evening.
A quick glance at the release notes for v1.118.0 didn't reveal anything too serious, though.

**Comment by ratti** [comment]
From the [release notes](https://github.com/immich-app/immich/discussions/13459):
> /api/server-info/* has been removed. Use /api/server/* instead.

When I execute the script:
`docker run -e API_URL="https://my.example.com/api/" -e API_KEY="MyAPI_key" -e ROOT_PATH="/e/Fotos/Ereignisse/public" -e ALBUM_LEVELS="-1"  salvoxia/immich-folder-album-creator:latest /script/immich_auto_album.sh
`

the output is:
```
time=2024-10-15T16:12:08.010+00:00 level=ERROR msg=Communication with Immich API failed! Make sure the passed API URL is correct!
time=2024-10-15T16:12:08.010+00:00 level=ERROR msg=Error in API call: {'message': 'Cannot GET /api/server-info/version', 'error': 'Not Found', 'statusCode': 404, 'correlationId': 'be6opc0e'}
Traceback (most recent call last):
  File "/script/immich_auto_album.py", line 866, in <module>
    version = fetchServerVersion()
              ^^^^^^^^^^^^^^^^^^^^
  File "/script/immich_auto_album.py", line 306, in fetchServerVersion
    checkApiResponse(r)
  File "/script/immich_auto_album.py", line 809, in checkApiResponse
    response.raise_for_status()
  File "/usr/local/lib/python3.12/site-packages/requests/models.py", line 1024, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://my.example.com/api/server-info/version
```

I'm not familiar with python, but that seems to be obvious. ;-)  `Cannot GET /api/server-info/version` has been removed.

**Comment by Felix-Franz** [comment]
I've created pull request #72 to fix this.
@Salvoxia can you please have a look at this pretty small change? 😄

**Comment by Salvoxia** [comment]
Thanks @Felix-Franz !
Released with [v0.14.0](https://github.com/Salvoxia/immich-folder-album-creator/releases/tag/0.14.0).
